pydantic/schema.py

```diff
@@ -849,32 +849,65 @@ def encode_default(dft: Any) -> Any:
 
 
 _map_types_constraint: Dict[Any, Callable[..., type]] = {int: conint, float: confloat, Decimal: condecimal}
-_field_constraints = {
-    'min_length',
-    'max_length',
-    'regex',
-    'gt',
-    'lt',
-    'ge',
-    'le',
-    'multiple_of',
-    'min_items',
-    'max_items',
+_field_constraints = {  # field constraints with the default value
```

Comment:
I understand why you added the default value considering Samuel's remark. But I feel like we put some logic at the wrong place. Couldn't we set this somewhere in `fields.py` and use it everywhere?
Maybe it can be done in a separate PR as we already used to have the field names here

Reply:
https://github.com/samuelcolvin/pydantic/pull/2196#discussion_r550939024